rust_src/src/keymap.rs

```diff
@@ -2,9 +2,9 @@
 
 use remacs_macros::lisp_fn;
 use remacs_sys::{current_global_map as _current_global_map, globals, EmacsInt, CHAR_META};
-use remacs_sys::{access_keymap, get_keymap, maybe_quit, Fevent_convert_list, Ffset, Fpurecopy,
-                 Fset};
-use remacs_sys::Qkeymap;
+use remacs_sys::{access_keymap, get_keymap, maybe_quit, Fcons, Fevent_convert_list, Ffset,
```

Comment:
In a future PR, would you mind pulling out the `Ffunc` group into its own `use` line?

Reply:
Ah. Noted. I shall do that, thanks!